Src/DependencyCollector/Shared/Implementation/DependencyTableStore.cs

```diff
@@ -35,6 +35,7 @@ internal static DependencyTableStore Instance
 
         public void Dispose()
         {
+            instance = null;
```

Comment:
There is an example of disposing of a static singleton that I found in the base SDK where it does this:
```
    if (disposing)
    {
        Interlocked.CompareExchange(ref instance, null, this);
    ...
```